src/test/java/pathfinding/ConcreteGraphTest.java

```diff
@@ -6,12 +6,10 @@
 
 import static org.junit.Assert.*;
 
-public class ConcreteGraphTest
-{
+public class ConcreteGraphTest {
```

Comment:
Wrong formatting, I'm going to work on getting some standardized code format that we can use on this project. Check your IntelliJ settings and change your braces placement + use tabs over spaces. I'll allow this an other formatting tweaks for now, but expect a commit from me fixing these soon.